server/static/js/config_settings.js

```diff
@@ -16,6 +16,8 @@ define (function() {
     'ext/mailcheck': 'ext/mailcheck.min',
     'ext/jqueryMigrate': 'ext/jquery-migrate-1.2.1',
     'ext/typeahead': 'ext/typeahead.bundle',
+    'ext/react': window.env == 'dev' ? 'ext/react-0.13.1' : 'ext/react-0.13.1.min',
```

Comment:
nit: pls use `===` everywhere (though it doesn't matter in this case since it's node, but good to be consistent)